heartbeat/docs/getting-started.asciidoc

```diff
@@ -1,184 +1,73 @@
-[[heartbeat-getting-started]]
-== Get started with {beatname_uc}
+[id="{beatname_lc}-installation-configuration"]
+== {beatname_uc} quick start: installation and configuration
 
 ++++
-<titleabbrev>Get started</titleabbrev>
+<titleabbrev>Quick start: installation and configuration</titleabbrev>
 ++++
 
-include::{libbeat-dir}/shared-getting-started-intro.asciidoc[]
+This guide describes how to get started quickly collecting uptime data about
+your hosts. You'll learn how to:
 
-* <<heartbeat-installation>>
-* <<heartbeat-configuration>>
-* <<heartbeat-template>>
-* <<load-kibana-dashboards>>
-* <<heartbeat-starting>>
-* <<view-kibana-dashboards>>
-* <<setup-repositories>>
+* install {beatname_uc}
+* specify the protocols to monitor
+* send uptime data to {es}
+* visualize the uptime data in {kib}
 
+// REVIEWERS: This screenshot comes from demo.elastic.co. Let me know if you
+// have a build or screenshot that you'd prefer to use here. 
 
-[[heartbeat-installation]]
+[role="screenshot"]
+image::./images/heartbeat-statistics.png[{beatname_uc} HTTP monitoring dashboard]
+
+[float]
+=== Before you begin
+
+You need {es} for storing and searching your data, and {kib} for visualizing and
+managing it.
+
+include::{libbeat-dir}/tab-widgets/spinup-stack-widget.asciidoc[]
```

Comment:
Side note: It would be awesome if these links could open in a new window

Reply:
Totally agree, though I think this would require a docs tooling change. We wouldn't want to change the behavior of Beats links without doing the same across the library. Maybe open a request in the docs repo? It's possible there's one already open. :-P